docs/docs/engines.md

```
@@ -71,6 +71,40 @@ duckdb.sql(f"""
 duckdb.sql("SELECT * FROM my_datalake.my_namespace.my_table").show()
 ```
 
+## <img src="/assets/firebolt.svg" width="30"> Firebolt
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)